geoAssembler/nb/tabs.py

```diff
@@ -189,49 +189,37 @@ def _sel_shape(self, selection):
                                   self.shape_drn]+self._create_spin_boxes(size, angle))
         self.children = [self.row1, self.row2]
 
-    def _move_quadrants(self, pos):
+    def _move_quadrants(self, _pos):
         """Shift a quadrant."""
-        d = pos['owner'].description.lower()[0]
-        pn = int(pos['new'])
-        po = int(pos['old'])
-        delta = pn - po
-        if d == 'h':
-            pos = np.array((delta, 0))
-            if self.parent.frontview:
-                pos = -pos
-        else:
-            pos = np.array((0, delta))
-        self.parent.geom.move_quad(self.parent.quad, pos)
+        offset = (self.posx_sel.value, self.posy_sel.value)
+        self.parent.geom.set_quad_offset(self.parent.quad, offset)
```

Comment:
This used to depend on some numpy calls, now that they're gone we can remove the numpy import on line 8